Automattic/wp-calypso
# Login: Handle users already logged in by redirecting them right away

This PR adds redirect when logged in user tries to reach `/login` page. Calypso Login page is only enabled for development env. As part of this PR I enabled server side rendering to improve user experience for users.

### Testing
1. Open http://calypso.localhost:3000/login when logged out.
2. Make sure it loads properly with code rendered server side.
3. Open http://calypso.localhost:3000/login when logged in.
4. Make sure it redirects client side to Reader (`/`).
5. Open http://calypso.localhost:3000/account-recovery when logged out.
6. Make sure it loads properly.
7. Open http://calypso.localhost:3000/account-recovery when logged in.
8. Make sure it redirects client side to Reader (`/`).

### Review
